index.html

```diff
@@ -4,9 +4,77 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@900&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/f8112dc1f2.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css">
   <title>My Portfolio</title>
 </head>
 <body>
-  
+  <header>
+    <nav>
+      <a class="wrap logo" href="#">Kehinde Audu</a>
+      <i class="fas fa-bars"></i>
```

Comment:
Please use the `nav` tag for the hamburger menu only.

Reply:
Hello Sajjad, please I'm new to this, could you explain what I need to do. I'm some how confused. Thanks for your understanding